test/org/opensolaris/opengrok/configuration/messages/StatsMessageTest.java

```diff
@@ -67,7 +61,7 @@ public void testValidate() {
         Assert.assertNull(m.getClassName());
         m.setTags(new TreeSet<>());
         Assert.assertTrue(MessageTest.assertValid(m));
-        Assert.assertEquals(new TreeSet<>(), m.getTags());
+        assertEquals(new TreeSet<>(), m.getTags());
```

Comment:
I dont know about this but we use I'd say in all of the tests the `Assert.assert ...` so you better change your Idea settings or redo the whole test style so it goes only one way.

Reply:
Why? Static import for asserts in tests is the most common static import I've ever seen. And the tests are much more readable.

Reply:
I'm not against but do it in the whole file so it's not mixed